client/lib/signup/site-type.js

```diff
@@ -4,14 +4,19 @@
 import i18n from 'i18n-calypso';
 import { find, get } from 'lodash';
 
+/**
+ * Internal Dependencies
+ */
+import config from '@automattic/calypso-config';
+
 /**
  * Internal dependencies
  */
 
 const getSiteTypePropertyDefaults = ( propertyKey ) =>
 	get(
 		{
-			theme: 'pub/hever',
+			theme: config.isEnabled( 'gutenboarding/site-editor' ) ? 'pub/seedlet-blocks' : 'pub/hever',
```

Comment:
I think we should now default to `blockbase`! 😅 

Also, any reasons to pick Hever for non-FSE sites?

cc @ianstewart for up-to-date opinions about default themes

Reply:
Thanks! Updated default theme

Reply:
Blockbase seems OK to me for a default theme for testing for now.

Reply:
> Also, any reasons to pick Hever for non-FSE sites?

We should match whatever "control" is using for non-FSE sites as a fallback. (Though I think we should not use Hever there anymore.)